types/d3-scale-chromatic/d3-scale-chromatic-tests.ts

```diff
@@ -120,24 +122,28 @@ const schemeReds: string = d3ScaleChromatic.schemeReds[3][0]; // #fee0d2
 // Sequential(Multi-Hue)
 // -----------------------------------------------------------------------
 
+colorInterpolator = d3ScaleChromatic.interpolateTurbo;
 colorInterpolator = d3ScaleChromatic.interpolateViridis;
-colorInterpolator = d3ScaleChromatic.interpolateMagma;
 colorInterpolator = d3ScaleChromatic.interpolateInferno;
+colorInterpolator = d3ScaleChromatic.interpolateMagma;
 colorInterpolator = d3ScaleChromatic.interpolatePlasma;
-colorInterpolator = d3ScaleChromatic.interpolateRainbow;
-colorInterpolator = d3ScaleChromatic.interpolateSinebow;
+colorInterpolator = d3ScaleChromatic.interpolateCividis;
 colorInterpolator = d3ScaleChromatic.interpolateWarm;
 colorInterpolator = d3ScaleChromatic.interpolateCool;
+colorInterpolator = d3ScaleChromatic.interpolateRainbow;
+colorInterpolator = d3ScaleChromatic.interpolateSinebow;
 colorInterpolator = d3ScaleChromatic.interpolateCubehelixDefault;
 
+const Turbo: string = d3ScaleChromatic.interpolateTurbo(0.7);
 const Viridis: string = d3ScaleChromatic.interpolateViridis(0.7);
-const Magma: string = d3ScaleChromatic.interpolateMagma(0.7);
 const Inferno: string = d3ScaleChromatic.interpolateInferno(0.7);
+const Magma: string = d3ScaleChromatic.interpolateMagma(0.7);
 const Plasma: string = d3ScaleChromatic.interpolatePlasma(0.7);
-const Rainbow: string = d3ScaleChromatic.interpolateRainbow(0.7);
-const Sinebow: string = d3ScaleChromatic.interpolateSinebow(0.7);
+const Cividis: string = d3ScaleChromatic.interpolateCividis(0.7);
 const Warm: string = d3ScaleChromatic.interpolateWarm(0.7);
 const Cool: string = d3ScaleChromatic.interpolateCool(0.7);
+const Rainbow: string = d3ScaleChromatic.interpolateRainbow(0.7);
+const Sinebow: string = d3ScaleChromatic.interpolateSinebow(0.7);
 const CubehelixDefault: string = d3ScaleChromatic.interpolateCubehelixDefault(0.7);
```

Comment:
I've re-ordered the interpolators in this section to match the type definitions and the README of `d3-scale-chromatic`.